project/Release.scala

```diff
@@ -27,14 +27,32 @@ object Release extends ParadoxKeys {
     val projectRef = extracted.get(thisProjectRef)
     val repo = extracted.get(Publish.defaultPublishTo)
     val state1 = extracted.runAggregated(publishSigned in projectRef, state)
-    // Make sure you set "-Dakka.genjavadoc.enabled=true" otherwise no
-    // japi will be generated and this following match will fail:
-    val (state2, Seq(japi, api)) = extracted.runTask(unidoc in Compile, state1)
-    val (state3, docs) = extracted.runTask(paradox in ProjectRef(projectRef.build, "akka-docs") in Compile, state2)
 
     IO.delete(release)
     IO.createDirectory(release)
     IO.copyDirectory(repo, release / "releases")
+    state1
+  }
+
+  def buildDocsCommand = Command.command("buildDocs") { state =>
+    if (!sys.props.contains("akka.genjavadoc.enabled"))
+      throw new RuntimeException("Make sure you start sbt with \"-J-Dakka.genjavadoc.enabled=true\" otherwise no japi will be generated")
```

Comment:
should be enough with `-D` instead of `-J-D`?

Reply:
Oh, didn't know, sure